tests/functional/DatadirTest.php

```diff
@@ -21,57 +21,41 @@ protected function getDataProviders(): array
         ];
     }
 
-    /**
-     * @dataProvider provideDatadirSpecifications
-     */
-    public function testDatadir(DatadirTestSpecificationInterface $specification): void
+    public function assertDirectoryContentsSame(string $expected, string $actual): void
     {
-        $tempDatadir = $this->getTempDatadir($specification);
+        $this->prettifyAllManifests($actual);
```

Comment:
Ako nativne typy sa pouzivali typy z Dibi (bude este komentovane), ... teraz sa pouzivaju nativne ODBC typy stlpcov.
Preto nastala zmena vo vsetkych manifestoch -> v datadir testoch.
Preto som manifesty dal rovno "prettyPrinted", kedze by aj tak boli v tomto PR vsetky -> tak nech je to aspon prehladnejsie.